logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -617,6 +618,11 @@ class UserSessionScope internal constructor(
 
     private val messageEncoder get() = MessageContentEncoder()
 
+    private val receiptMessageHandler = ReceiptMessageHandler(
```

Comment:
Any reason not have this as a  `get() = `?

Reply:
Not really. Just no need to have a singleton for it as well.

Considering ApplicationMessageHandler is the only entity that uses it, and it is a singleton, the end result should be the same:

- A single instance of it per user scope.

Reply:
I'll change it to `get() =`